frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -95,6 +97,20 @@ function mapStateToProps(state: State) {
   };
 }
 
+const EMBED_THEME_CLASSES = (theme: HashOptions["theme"]) => {
+  if (!theme) {
+    return null;
+  }
+
+  if (theme === "night") {
+    return cx(ParameterValueWidgetS.ThemeNight, EmbedFrameS.ThemeNight);
+  }
+
+  if (theme === "transparent") {
+    return EmbedFrameS.ThemeTransparent;
+  }
+};
```

Comment:
I tried to look if we could use just a map but typescript was a bit annoyed as we used `undefined` if we want the default theme, I don't have a strong preference but let's see what others say
```diff
diff --git a/frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx b/frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx
index f2ad14666a..dfb3fb3638 100644
--- a/frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx
+++ b/frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx
@@ -97,18 +97,10 @@ function mapStateToProps(state: State) {
   };
 }
 
-const EMBED_THEME_CLASSES = (theme: HashOptions["theme"]) => {
-  if (!theme) {
-    return null;
-  }
-
-  if (theme === "night") {
-    return cx(ParameterValueWidgetS.ThemeNight, EmbedFrameS.ThemeNight);
-  }
-
-  if (theme === "transparent") {
-    return EmbedFrameS.ThemeTransparent;
-  }
+const EMBED_THEME_CLASSES = {
+  night: cx(ParameterValueWidgetS.ThemeNight, EmbedFrameS.ThemeNight),
+  transparent: EmbedFrameS.ThemeTransparent,
+  default: "",
 };
 
 function EmbedFrame({
@@ -194,7 +186,7 @@ function EmbedFrame({
       className={cx(
         EmbedFrameS.EmbedFrame,
         className,
-        EMBED_THEME_CLASSES(theme),
+        EMBED_THEME_CLASSES[theme ?? "default"],
       )}
       data-testid="embed-frame"
       data-embed-theme={theme}

```

Reply:
Yeah, I think I would have used a map if there were more themes but for now I felt it just made more sense to keep it like this. I think if we decide to add more themes in the future, we should split this out into a different file and do a map thing